srtcore/list.cpp

```diff
@@ -123,7 +123,16 @@ int CSndLossList::insert(int32_t seqno1, int32_t seqno2)
     // Find the insert position in the non-empty list
     const int origlen = m_iLength;
     const int offset  = CSeqNo::seqoff(m_caSeq[m_iHead].seqstart, seqno1);
-    int       loc     = (m_iHead + offset + m_iSize) % m_iSize;
+
+    if (offset >= m_iSize)
```

Comment:
this check doesn't cover case with seqno2!= seqno1.
E.g is offset < m_iSize but (offset + CSeqNo::seqlen(seqno1, seqno2)) >= m_iSize we should probably
fix seqno2 or fail CSndLossList::insert

Reply:
Good point! Inserting a longer range should not produce any issue, but logically should be limited.
At the same time, additional validation of `seqno2` might slightly impact the performance of this `insert` call. 🤔

Reply:
Ignore this finding then. As I understand we need this quick fix for 1.4.*, but later we will have #1012 finally integrated so we don't have to worry to much about such minor issues.
Please reply my comment below about m_iFlowWindowSize. I think it is more important.

Reply:
#1012 has to be reworked a bit. All entries must be stored in order in the sender loss list. In that PR the `insert` function tries to loop over the container and find an empty place to store a new entry.